### ISSUE TITLE
Missing tray icon causes ArmCord to launch twice

### ISSUE THREAD

**Issue by takase1121** [smartfrigde/armcord#458]
**Describe the bug**

When tray icon is not available, ArmCord can be launched multiple times, causing some database issues.

**To Reproduce**

1. Get a system that doesn't have a tray icon (e.g. sway + yambar).
2. Launch ArmCord, it should work fine.
3. Launch ArmCord again, it should complain about database lock error (and RPC port already used error), and opens up to the Discord login page.
4. Kill both ArmCord instances with `pkill electron`.

**Expected behavior**
When ArmCord can't create a tray icon, the close button automatically closes ArmCord insteads of trying to minimize into tray.

**Screenshots**
![20230907_18h56m08s_grim](https://github.com/ArmCord/ArmCord/assets/20792268/1f3049da-538f-4871-a828-745af3d093fa)

**Desktop (please complete the following information):**
 - OS: Arch Linux 6.4.12-zen1-1-zen, labwc 0.6.4, yambar 1.10.0-7-g9a111a5
 - Method of installation: AUR
 - Version: armcord-git r978.d68b1db-1

**Additional context**

This is not my daily driver, its an environment I use to preserve RAM. I don't want to disable tray icons completely because it is available in KDE.

**Comment by RandomAnimeGamer** [comment]
As a temporary workaround you can set the app to not minimize to the tray in your settings, but if there is no tray icon that should always be set to not minimize to tray.

**Comment by orowith2os** [comment]
> but if there is no tray icon that should always be set to not minimize to tray.

A tray icon is not necessary in order to rely on the app not fully closing when you close its windows.

**Comment by takase1121** [comment]
> > but if there is no tray icon that should always be set to not minimize to tray.
> 
> A tray icon is not necessary in order to rely on the app not fully closing when you close its windows.

I understand this. Either ArmCord doesn't allow it to be opened twice, or it doesn't allow minimize to tray icon when tray isn't available. I'm not sure which approach is easier, but both would solve the issue.

**Comment by takase1121** [comment]
I did some investigation; turns out by default ArmCord doesn't enable multi-instance, but somehow (proabbly due to config transition) multi-instance is set to undefined, so the ArmCord settings shows it as false, but `main.ts` doesn't recognize it? I removed `.config/ArmCord` and can't replicate the issue anymore.

Now, even when tray isn't available, it won't run ArmCord twice. I consider this as solved.
